README.md

```diff
@@ -91,5 +91,6 @@ Botpress Partners is a list of agencies who can help you build your next convers
 | [Tasqat](https://www.tasqat.com)| Dubai, United Arab Emirates |
 | [Creative Melon](https://creativemelon.co.za) | Johannesburg, South Africa |
 | [PaperGo](https://www.papergo.io) | Patras, Greece |
+| [BotArtisanz](http://botartisanz.com/) | India, Kerala |
```

Comment:
According to other entries, I guess it should instead be

    Kerala, India

Reply:
Yes,